ts/src/header-validator/source.test.ts

```diff
@@ -39,7 +39,16 @@ const testCases: TestCase[] = [
       "filter_data": {"b": ["c"]},
       "priority": "2",
       "source_event_id": "3",
-      "max_event_level_reports": 2
+      "max_event_level_reports": 2,
+      "aggregatable_debug_reporting": {
+        "budget": 1234,
+        "key_piece": "0x2",
+        "debug_data": [ {
+          "key_piece": "0x1",
+          "types": ["source-success"],
```

Comment:
Let's also add an unknown debug data type here to ensure that it roundtrips properly.